Makefile

```diff
@@ -187,10 +187,16 @@ clean-target-dir:
 
 # Dependencies
 
-.venv/bin/python: Makefile
+.venv/bin/python:
```

Comment:
I didn't really get why the `Makefile` prerequisite...

Do we use `make` to generate the `Makefile` under any env?
Is this some trick I have not learned?

Reply:
That was because we install packages into the virtualenv. When the makefile changes, then also the dependencies might change. Please leave that in, since it really has no disadvantage.